cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/preview/DefaultPreviewRunner.java

```diff
@@ -54,20 +54,27 @@
 import com.google.gson.JsonElement;
 import com.google.inject.Inject;
 import org.apache.twill.common.Threads;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.Timer;
+import java.util.TimerTask;
 import javax.annotation.Nullable;
 
 /**
  * Default implementation of the {@link PreviewRunner}.
  */
 public class DefaultPreviewRunner extends AbstractIdleService implements PreviewRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(DefaultPreviewRunner.class);
   private static final Gson GSON = new Gson();
+  // default to 10 mins
+  private static final int DEFAULT_TIMEOUT = 10;
```

Comment:
We keep injector in memory for 15 minutes, right? 
Should we have same config for this one too and make it configurable?